packages/gatsby-cli/src/create-cli.ts

```diff
@@ -534,6 +508,95 @@ Creating a plugin:
         report.log(`Telemetry collection ${enabled ? `enabled` : `disabled`}`)
       }),
     })
+    .command({
+      command: `plugin [cmd]`,
+      describe: `Useful commands relating to Gatsby plugins`,
+      builder: yargs =>
+        yargs.positional(`cmd`, {
+          choices: [`docs`],
+          describe: "Valid commands include `docs`.",
+          type: `string`,
+        }),
+      handler: handlerP(({ cmd }) => {
+        if (cmd === `docs`) {
+          console.log(`
+      Using a plugin:
+      - What is a Plugin? (https://www.gatsbyjs.org/docs/what-is-a-plugin/)
+      - Using a Plugin in Your Site (https://www.gatsbyjs.org/docs/using-a-plugin-in-your-site/)
+      - What You Don't Need Plugins For (https://www.gatsbyjs.org/docs/what-you-dont-need-plugins-for/)
+      - Loading Plugins from Your Local Plugins Folder (https://www.gatsbyjs.org/docs/loading-plugins-from-your-local-plugins-folder/)
+      - Plugin Library (https://www.gatsbyjs.org/plugins/)
+
+      Creating a plugin:
+      - Naming a Plugin (https://www.gatsbyjs.org/docs/naming-a-plugin/)
+      - Files Gatsby Looks for in a Plugin (https://www.gatsbyjs.org/docs/files-gatsby-looks-for-in-a-plugin/)
+      - Creating a Generic Plugin (https://www.gatsbyjs.org/docs/creating-a-generic-plugin/)
+      - Creating a Local Plugin (https://www.gatsbyjs.org/docs/creating-a-local-plugin/)
+      - Creating a Source Plugin (https://www.gatsbyjs.org/docs/creating-a-source-plugin/)
+      - Creating a Transformer Plugin (https://www.gatsbyjs.org/docs/creating-a-transformer-plugin/)
+      - Submit to Plugin Library (https://www.gatsbyjs.org/contributing/submit-to-plugin-library/)
+      - Source Plugin Tutorial (https://www.gatsbyjs.org/tutorial/source-plugin-tutorial/)
+      - Maintaining a Plugin (https://www.gatsbyjs.org/docs/maintaining-a-plugin/)
+      - Join Discord #plugin-authoring channel to ask questions! (https://gatsby.dev/discord/)
+                 `)
+        } else {
+          console.log(`Current valid subcommands are: 'docs'`) // right now this is hardcoded because we only have a single command
+        }
+      }),
+    })
+    .command({
+      command: `config [cmd] [key] [value]`,
+      describe: `View or set your gatsby-cli configuration settings.`,
```

Comment:
Question here - won't `gatsby config` command cause confussion with `gatsby-config` (the file) and both of them dealing with different things? I get the use of `config` command in cli here as many CLIs use similar conventions, just feel like in our case we might need different name as we already have "gatsby config" around.

Reply:
Ya, I had the same concern. There is a bit of a bikeshed in slack about it, but certainly open to further discussion. Simple enough to change.

Reply:
Can't find the bikeshed thread, but certainly `gatsby options` came up, didn't it?

Reply:
I don’t recall that it did, but I like it and it’s not confusing. So let’s go with that!